application/Core/CLI/Output/IConsoleFormatter.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+namespace MORR.Core.CLI.Output
+{
+    /// <summary>
+    /// The IConsoleFormatter abstracts from the default
+    /// Console output allowing to insert different behaviors on printing
+    /// and a hierarchical printing options.
+    /// </summary>
+    public interface IConsoleFormatter
```

Comment:
We might have found use for a similar interface for logging purposes (same set of methods, just without `Read()`), but this is out of scope here as such an Interace would have to reside outside of the CLI package.

Reply:
Is this fine for you if I merge it with the interface included in the CLI?

Reply:
Should be fine if that makes it easier, i would suggest doing that in a new PR however.